Testing/WinMLRunnerTest/WinMLRunnerTest.vcxproj

```diff
@@ -204,6 +204,17 @@
       <DeploymentContent Condition="'$(Configuration)|$(Platform)'=='Release|x64'">true</DeploymentContent>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\..\SharedContent\models\Add_ImageNet224.onnx">
```

Comment:
I found out that the Add_ImageNet1920.onnx file asked in one test was not even in the repo, so the test wasn't really accurate. I changed the test to use Add_ImageNet224.onnx since I couldn't find the original